core/lib/dal/migrations/20240522081114_create_data_availability_table.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE data_availability
```

Comment:
It's probably safer to use `IF NOT EXISTS` here and `IF EXISTS` in the revert migration just in case. cc @Deniallugo @perekopskiy

Reply:
added